starter-project-web-vue2/components/liben/LibenArticleForm.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <form>
+    <v-text-field
+      v-model="title"
+      :error-messages="titleErrors"
+      label="Title"
+      required
+      @input="$v.title.$touch()"
+      @blur="$v.title.$touch()"
+    ></v-text-field>
+
+    <v-text-field
+      v-model="description"
+      :error-messages="descriptionErrors"
+      label="Description"
+      required
+      @input="$v.description.$touch()"
+      @blur="$v.description.$touch()"
+    ></v-text-field>
+
+    <v-textarea
+      v-model="content"
+      :error-messages="contentErrors"
+      label="Content"
```

Comment:
This is one way of doing it.
The cleaner way is using [rules](https://vuetifyjs.com/en/components/forms/#props) from vuetify.

Reply:
fixed, I should have considered that